guides/common/attributes.adoc

```diff
@@ -44,6 +45,10 @@ endif::[]
 ifeval::["{build}" == "satellite"]
 :satellite:
 endif::[]
+ifdef::katello[]
```

Comment:
This isn't working for me, not serving new names. 
I think the Katello build needs a new name.